params_and_main.py

```diff
@@ -88,13 +88,22 @@
 large_file = False # If predicted probabilities should be stretched to int8 to increase storage capacity
 max_empty = 0.2  # Maximum no data area in created image crops
 ARCHITECTURE = xresnet34 #xresnet34
-transforms = [Dihedral(0.5),  # Horizontal and vertical flip
-              Rotate(max_deg=180, p=0.5),  # Rotation in any direction possible
-              Brightness(0.2, p=0.5),
-              Contrast(0.2, p=0.5),
-              Saturation(0.2),
-              Normalize.from_stats(*imagenet_stats)]
-transforms = None
+aug_pipe = A.Compose([
+            A.HorizontalFlip(p=0.5), # Applies a horizontal flip to the image with a probability of 0.5.
+            A.VerticalFlip(p=0.5), # Applies a vertical flip to the image with a probability of 0.5.
+            A.ShiftScaleRotate(p=0.5), # Randomly applies affine transforms: translation, scaling and rotation in one call with a probability of 0.5.
+            A.RandomBrightnessContrast( # Randomly changes brightness and contrast of the image with a probability of 0.5.
+                brightness_limit=(-0.1,0.1), 
+                contrast_limit=(-0.1, 0.1), 
+                p=0.5
+            ),
+            A.CoarseDropout(p=0.5), # Randomly masks out rectangular regions in the image with a probability of 0.5.
+
+]) # For more Augmentation options: https://github.com/albumentations-team/albumentations/tree/main#i-am-new-to-image-augmentation
+
+Num = 2 # Number of the augmented images minus from the totla number of batch size (default is 2).
```

Comment:
i think this means that 2 images within a batch gonna be not transformed - is this meant to be? Maybe i got this wrong but i thought we wanted to transform all images. With batchsize 4 there will be 50 % of images transformed.
 
### see albumentation function (utils.py, line  #327):  ##
Leave the second proportion of the batch unchanged
        for img, mask in zip(batch_img[int(self.Num - len(batch_img)):], batch_mask[int(self.Num - len(batch_img)):]):

Reply:
good idea Benny, I have already changed this line in the function:

for img, mask in zip(batch_img[int(self.n_transform_imgs - len(batch_img)):], batch_mask[int(self.n_transform_imgs - len(batch_img)):]):

So now the number that the user will define as  "n_transformed_images" will be the specific number for augmented!